release-note-script/src/main/java/MergeReleaseNotes.java

```diff
@@ -0,0 +1,242 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * MergeReleaseNotes creates the body of the merged release note for
+ * ScalarDB. This script takes release note bodies for ScalarDB, ScalarDB
+ * Cluster, ScalarDB GraphQL, and ScalarDB SQL as markdown files as its
+ * input. And then output the merged release note body to standard
+ * output as a markdown.
+ * 
+ * This script is assumed to be executed in a GitHub Actions workflow.
+ * 
+ * Note that it is needed that java 11 to execute this script since it is
+ * executed as a Single-File Source-code program.
+ */
+public class MergeReleaseNotes {
```

Comment:
It looks like the fields are defined with `private` but the methods are defined with `package private`. I think we should unify them?

Reply:
Thank you for the point it out.
I reorganized the members to unify the scope of them. Basically, I set the fields as private, and methods that do not need to be accessed from outside the class as private.